pkgs/misc/vscode-extensions/ms-dotnettools-csharp/default.nix

```
@@ -0,0 +1,142 @@
+{ lib
+, fetchurl
```

```suggestion
{ fetchurl
```

Please don't use stdenv and lib together.